### PR TITLE
Changelog for search tokenizer

### PR DESCRIPTION
**Pull request by PreciselyAlyss**
@cbetta changelog for search improvements from January

### CODE REVIEW

File: content/pages/changelog/index.md [hunk 1]
```diff
@@ -13,6 +13,19 @@ For historical changelog entries, please see our
 [2019](page://changelog/2019) and
 [2018 release notes](page://changelog/2018).
 
+## 2020-07-20 / Updates to Search Relevance & Performance
+
+In January, updates were released to Box's search [tokenizer](wiki-tokenizer),
```
Comment (PreciselyAlyss) [inline content/pages/changelog/index.md:18]: @cbetta wording may be a bit awkward here. Feel free to use GitHub Suggestions feature so we can just commit from the web experience.

**Comment by PreciselyAlyss** [comment]
Failure from linter because of the words `tokenizer` and `tokenized` 😿

File: content/pages/changelog/index.md (hunk 1 above)
Comment (cbetta) [reply content/pages/changelog/index.md:18]: Looks fine to me tbh.

File: content/pages/changelog/index.md [hunk 2]
```diff
@@ -13,6 +13,19 @@ For historical changelog entries, please see our
...
+improving the relevancy of search results and performance. This change does not
+effect how metadata is tokenized.
+
+Notable changes to the API-initiated searches include:
```
Comment (cbetta) [inline content/pages/changelog/index.md:22]: Are these changes API specific?

Comment (cbetta) [reply content/pages/changelog/index.md:22]: If this is not API specific then I'd rather say.

```suggestion
Notable changes to searches include:
```

File: content/pages/changelog/index.md [hunk 3]
```diff
@@ -13,6 +13,19 @@ For historical changelog entries, please see our
...
+* Ability to use "[stop words](wiki-stop-words)" (e.g. to do list)
```
Comment (cbetta) [inline content/pages/changelog/index.md:23]:
```suggestion
* The ability to use "[stop words](wiki-stop-words)" (e.g. to do list)
```

File: content/pages/changelog/index.md [hunk 4]
```diff
@@ -13,6 +13,19 @@ For historical changelog entries, please see our
...
+* Improved Japanese search
```
Comment (cbetta) [inline content/pages/changelog/index.md:24]: Can you remind me how Japanese search was improved? Like the tokenization is more accurate or the way a search is resolves has been improved?

File: content/pages/changelog/index.md (hunk 2 above)
Comment (PreciselyAlyss) [reply content/pages/changelog/index.md:22]: it isn't API specific. I was just at a loss for how to represent that some of the changes to the webapp search experience wouldn't be reflected in the API

File: content/pages/changelog/index.md (hunk 4 above)
Comment (PreciselyAlyss) [reply content/pages/changelog/index.md:24]: I'll get more details from Rebecca. The DDOC just had a link to the community post so I borrowed the language from that.

File: content/pages/changelog/index.md (hunk 2 above)
Comment (cbetta) [reply content/pages/changelog/index.md:22]: My guess is we're not covering the web-app changes here, just the API. So it's kind of self explanatory. Happy to keep as is or just simplify.

**Pull request merged**